store/store.go

```diff
@@ -454,6 +454,7 @@ type FileInfoStore interface {
 	GetForUser(userId string) StoreChannel
 	InvalidateFileInfosForPostCache(postId string)
 	AttachToPost(fileId string, postId string) StoreChannel
+	AttachMultipleToPost(fileIds []string, postId string, creatorId string) StoreChannel
```

Comment:
1/5 Typographically `AttachToPostMultiple` looks  better and more consistent with vvv `PermanentDelete[Batch]`, but I don't know if we have a convention. Consistency > typography.